composer.json

```diff
@@ -15,9 +15,9 @@
     ],
     "require": {
         "php": "^5.3.6 || >=7.0 <7.2",
-        "composer/semver": "^1.4",
         "ext-json": "*",
         "ext-tokenizer": "*",
+        "composer/semver": "^1.4",
```

Comment:
admit it, you did add this dep originally by editing the file, not via command, didn't you ?

Reply:
> admit it, you did add this dep originally by editing the file, not via command, didn't you ?

asking cause ` "sort-packages": true` should've prevented the insert at the previous location? ;)
anyway, I edit `composer.json` by hand most of the time as well : P

Reply:
;)